apps/speed.c

```diff
@@ -561,52 +639,514 @@ static OPT_PAIR ecdh_choices[] = {
 };
 #endif
 
+#ifndef SIGALRM
+# define COND(d) (count < (d))
+# define COUNT(d) (d)
+#else
+# define COND(c) (run && count<0x7fffffff)
+# define COUNT(d) (count)
+#endif                         /* SIGALRM */
+
+static unsigned char *buf = NULL, *buf2 = NULL;
+static int j;
```

Comment:
Yes, I agree that the name is not descriptive at all.
On the other hand this variable is used very often also as an index for array, so I would not recommend a very long name.

`j` is used as an index for the different test cases of the operations, e.g. the lengths of the buffers for the ciphers.


Could we call it `tc` (as in **t**est **c**ase) and add a comment close to the declaration?
Any other suggestion is very welcome.